ttnn/cpp/ttnn/operations/eltwise/binary_backward/device/binary_backward_op.cpp

```diff
@@ -271,21 +271,13 @@ std::vector<std::optional<Tensor>> _eq_bw(
     std::vector<std::optional<Tensor>> result;
 
     if (are_required_outputs.at(0)) {
-        if(input_grad.has_value()){
-            tt::tt_metal::zeros_like(cq_id, input, output_mem_config, input_grad);
-        } else {
-            input_grad = tt::tt_metal::zeros_like(cq_id, input, output_mem_config);
-        }
+        input_grad = ttnn::operations::creation::zeros_like(input, input.get_dtype(), input.get_layout(), std::nullopt, output_mem_config);
```

Comment:
Can you clarify this change please? I am not sure why

Reply:
Is it because `    if (are_required_outputs.at(0)) {` this check is enough?

Reply:
It is because ttnn.zeros_like doesn't have queue_id support while tt_eager zeros_like had.